modules/product/src/Form/AddToCartForm.php

```diff
@@ -160,17 +189,14 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
   }
 
   /**
-   * Builds the form elements for selecting a variation.
+   * Create variation selection widget.
```

Comment:
Why change the phpdoc to one that doesn't conform to standards (s/Create/Creates, etc)